helm/grafana/templates/postgresql-cluster.yaml

```diff
@@ -0,0 +1,122 @@
+{{- if .Values.grafanaDatabase.enabled }}
+# Postgresql cluster
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: {{ .Chart.Name }}-postgresql
+  namespace: {{ .Release.Namespace }}
+spec:
+  instances: 2
+  storage:
+  size: 1Gi
+  backup:
+    barmanObjectStore:
+      data:
+        compression: gzip
+      wal:
+        compression: gzip
+      {{- if (eq .Values.grafanaDatabase.provider "capa") }}
+      destinationPath: s3://giantswarm-{{ .Values.grafanaDatabase.baseUrl }}-{{ .Chart.Name }}-postgresql/
+      s3Credentials:
+        inheritFromIAMRole: true
+      {{- end }}
+      {{- if (eq .Values.grafanaDatabase.provider "capz") }}
```

Comment:
This should be roughtly hte same for vsphere and cloud-directory I think

Reply:
Yes, that's why I only distinguish bewteen capa and capz in this chart. If it's not capa, the object storage will be in azure whatever the provider. The difference will be defined in the shared configs.